config/locales/user_mailer/en.yml

```diff
@@ -34,9 +34,9 @@ en:
         If you did not request a new account or suspect an error, please visit
         the %{app} %{help_link} or %{contact_link}.
       intro: >
-        Your request to create a %{app} account using this email address cannot
-        be granted because this email address is already in use. If you requested this
-        change, please use the following link to log in to %{app}:
+        This email address is already associated with a %{app} account, so we cannot create a new account with it. 
+        If you are not trying to sign in with this email address, you can ignore this message.
+        To sign in with your existing account, follow the link below. 
       link_text: Go to %{app}
       reset_password: >
```

Comment:
another thing I noticed when I got this email is that the `reset_password` does not actually link me to the reset password page. Should we do that?